scripts/python/PyAutoscoper/connect.py

```diff
@@ -103,6 +106,30 @@ def _openConnection(self):
         s.connect((self.address, 30007))
         return s
 
+    def _checkVersion(self):
+        """
+        Internal function, should not be called by a user.
+
+        Checks that the server version is compatible with the client version.
+
+        Called automatically upon init.
+
+        :raises AutoscoperServerError: If the server version is not compatible with the client version
+        """
+        response = self._send_command(0x10)  # 16
+        server_version = float(response[1:].decode("utf-8").strip("\x00"))
+
+        if self.verbose:
+            print(f"Server version: {server_version}")
+            print(f"Client version: {REQUIRED_SERVER_VERSION}")
+
+        if server_version == REQUIRED_SERVER_VERSION:
```

Comment:
The version checking seems incorrect, I would expected the client to work as expected if the version matches.

Reply:
Yes, looks like I forgot to negate this statement during a refactor where I inverted the contents of the if statement